### PR TITLE
Ios bugfix sticky and slider neighbor

### PR DESCRIPTION
**Pull request by ZhaoXigua**
fix attribute ‘sticky’ bug while scrolling up and down quickly && and SliderNeighbor component’s guesture-blocking bug when it is used in a scrollview

### CODE REVIEW

**Comment by acton393** [comment]
hi @ZhaoXigua  please sign the CLA (Contribution License Agreement) above

File: ios/sdk/WeexSDK/Sources/Component/WXSliderNeighborComponent.m [hunk 1]
```diff
@@ -474,6 +474,18 @@ - (id)viewOrSuperview:(UIView *)view ofClass:(Class)class
     return [self viewOrSuperview:view.superview ofClass:class];
 }
 
+-(id)viewInChain:(UIView *)view ofClass:(Class)class{
```
Comment (acton393) [inline ios/sdk/WeexSDK/Sources/Component/WXSliderNeighborComponent.m]: 这个函数哪个地方有用到

File: ios/sdk/WeexSDK/Sources/Component/WXSliderNeighborComponent.m [hunk 2]
```diff
@@ -514,6 +526,23 @@ - (BOOL)gestureRecognizer:(UIGestureRecognizer *)gesture shouldReceiveTouch:(UIT
     return YES;
 }
 
+- (BOOL)gestureRecognizer:(UIGestureRecognizer *)gestureRecognizer shouldRecognizeSimultaneouslyWithGestureRecognizer:(UIGestureRecognizer *)otherGestureRecognizer{
+    if ([gestureRecognizer isKindOfClass:[UIPanGestureRecognizer class]] && [otherGestureRecognizer isKindOfClass:[UIPanGestureRecognizer class]]) {
+        if ([otherGestureRecognizer.view isKindOfClass:[UIScrollView class]]) {
+            UIScrollView* scrollview = (UIScrollView *)otherGestureRecognizer.view;
+            if (scrollview.scrollEnabled) {
+                UIPanGestureRecognizer* panRcgn= (UIPanGestureRecognizer *)gestureRecognizer;
+                if (fabs([panRcgn translationInView:panRcgn.view].y) > fabs([panRcgn translationInView:panRcgn.view].x)*16) {
```
Comment (acton393) [inline ios/sdk/WeexSDK/Sources/Component/WXSliderNeighborComponent.m]: 这个地方加一下注释吧， 在什么情况允许手势同时识别

Comment (ZhaoXigua) [inline ios/sdk/WeexSDK/Sources/Component/WXSliderNeighborComponent.m]: if the view which the otherGestureRecognizer works on is a scrollview and also it is scrollEnabled ,at this time,we should not block the guesture from being recognized by the otherGestureRecognizer

**Pull request merged**